src/resources/views/crud/columns/summernote.blade.php

```diff
@@ -0,0 +1,28 @@
+{{-- regular object attribute --}}
```

Comment:
Wouldn't this just output the markdown in plain text? I don't see this column converting string to markdown, which I imagine is what's appropriate in this scenario. Can you please explain what you had in mind with this column type @munjaldevelopment ?

Reply:
Ping @munjaldevelopment - I think this still needs attention. See my comment above.

Reply:
I have applied changed as suggested. Am I still missing anything?